app/src/main/java/com/simplecity/amp_library/ui/fragments/LibraryController.java

```diff
@@ -122,9 +124,9 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
 
         ((AppCompatActivity) getActivity()).setSupportActionBar(toolbar);
 
-        setupViewPager();
+        RxBroadcast.fromLocalBroadcast(getContext(), new IntentFilter(EVENT_TABS_CHANGED)).subscribe(onNext -> {refreshPager = true;});
```

Comment:
A disposable must be added to the CompositeDisposable so it will be disposed of in `onDestroyView` (see `compositeDisposable.clear();`)